example/gtk/gtk_main.c

```diff
@@ -0,0 +1,168 @@
+#include <gtk/gtk.h>
+#include <gtksourceview/gtksource.h>
+#include <microtexequation.h>
+
+#include "mt_gtk_res.h"
+#include "isamples.h"
+
+static gchar* samples_file = NULL;
+
+struct _MicroTexGtkDemoWin {
+	GtkApplicationWindow parent_instance;
```

Comment:
Perhaps we shall use 2 white-space indents? I use CLion and VSCode personally (with clang-format), and it automatically formats the files 😂 , I think it is better to use the same code style (I didn't mean the tab indents are bad).

Reply:
I prefer using tab as proper indentation a lot, but sure.